src/main/resources/latest-spark-swagger-output.yaml

```diff
@@ -34,6 +34,9 @@ tags:
 - name: "otp"
   description: "Proxy interface for OTP endpoints. Refer to <a href='http://otp-docs.ibi-transit.com/api/index.html'>OTP's\
     \ API documentation</a> for OTP's supported API resources."
+- name: "otp2"
+  description: "Proxy interface for OTP endpoints. Refer to <a href='http://otp-docs.ibi-transit.com/api/index.html'>OTP's\
```

Comment:
I think OTP v2 should be mentioned in the description.

Reply:
Good point.